Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM sonarsource/sonar-scanner-cli:11.1
 
-LABEL version="3.0.0" \
+LABEL version="3.1.0" \
```

Comment:
I am assuming the version of the docker container should be 1-to-1 with the tag at release. Is that correct?

Reply:
yes